app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  #  :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
```

Comment:
- [ ] As mentioned by the previous reviewer, your application still raising the same error, while signing up `information for confirmation are not sent to the user`. According to the **[image](https://user-images.githubusercontent.com/71219070/218063249-60e810c2-813d-4fb9-96b5-21c5f86385db.PNG)** you left above, your application should behaviors like that but on my side, below is the result for **signing up action** I am still getting after trying many times:
![Screenshot from 2023-02-10 12-21-16](https://user-images.githubusercontent.com/83042742/218068934-76efd613-e60c-42cf-b539-3f777aa7cdfa.png)

Reply:
remove validation to fix error